src/cmd/wg_key.rs

```diff
@@ -0,0 +1,14 @@
+use crate::cmd::{Cmd};
+use crate::types::{WgPubkey};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct RegisterWgKey {
+    public_key: WgPubkey,
+}
+
+impl Cmd for RegisterWgKey {
+    type Output = ();
```

Comment:
Have we checked that upgrading `()` to a struct is backwards compatible? If not we should return an empty struct here in case we want to return more info in the future.

Reply:
Yes, we don't parse the response body for `()` at all, so they are compatible with any future type change.